cyclus/system.py

```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
 
 @lazyobject
```

Comment:
All of these lazyobject decorators will need to be removed. All of the functions here that are decorated with lazyobject will need to be rewritten.